NuGet.Config

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+    
+  <packageSources>
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
+    <add key="github-hackney" value="https://nuget.pkg.github.com/LBHackney-IT/index.json" />
+  </packageSources>
+
+  <packageSourceCredentials>
+    <github-hackney>
+      <add key="Username" value="PublicToken" />
+      <add key="ClearTextPassword" value= "%LBHPACKAGESTOKEN%" />
```

Comment:
I don't know if it's an issue within XML, but you have a whitespace after `value=`.

Reply:
ooh, this sounds promising